packages/dashboards/src/lib/components/timeseries-widget/chart-presets/xy-chart/xy-chart.component.ts

```diff
@@ -85,6 +90,28 @@ export abstract class XYChartComponent extends TimeseriesChartComponent
         }
     }
 
+    public onInteraction(legendSeries: IChartAssistSeries<IAccessors>, event: MouseEvent) {
+        let interactive = this.isSeriesInteractive(legendSeries);
```

Comment:
The result of this query may be too broad. What if there's a secondary link and no primary link, and the primary description is clicked? I think the event would be sent regardless. But, in that case, I think we should avoid emitting the event.